src/components/dropdown.tsx

```diff
@@ -0,0 +1,21 @@
+import { SupportedExportTypes } from './export-type';
+import GroupDropdown from './group-dropdown';
+import PatientDropdown from './patient-dropdown';
+
+export interface DropdownProps {
+  dropdownType: SupportedExportTypes;
+  getQueryIdFromDropdown: (id: string | null) => void;
+}
+
+/*
+ * Wrapper component for each type of dropdown associated with each export type
+ */
+export default function Dropdown({ dropdownType, getQueryIdFromDropdown }: DropdownProps) {
```

Comment:
So honestly, I would argue that this component isn't totally necessary! What are your thoughts on just calling PatientDropdown or GroupDropdown directly in `export-type.tsx` rather than this component with the dropdownType param? Basically, a conditional would go in export-type.tsx instead and that would eliminate the need for this file!

Reply:
Potentially made a fix to this. Let me know if you like this new way more.